### ISSUE TITLE
Muter stopped working

### ISSUE THREAD

**Issue by bwestover** [mastahyeti/OctoGAS#17]
Since 5/29 I have been getting a daily email from apps-scripts-notifications that says something like this:

```
Your script, OctoGAS, has recently failed to finish successfully. A summary of the failure(s) is shown below. To configure the triggers for this script, or change your setting for receiving future failure notifications, click here.

Details:

Start   Function    Error Message   Trigger End
6/11/16 10:32 PM    muter   Timeout: https://github.com/notifications/unsubscribe/<redacted> (line 59, file "muter", project "OctoGAS") time-based  6/11/16 10:33 PM
Sincerely,

Google Apps Script

Need help? Visit the Google Apps Script documentation. Please do not reply to this message. (c) 2016 Google
```

Sometimes the error is "Timeout: <url>" and other times it is "Address unavailable: <url>"
The list does not include every thread that I have attempted to mute on that day, but there are more failures in the list the more threads I have muted, suggesting some percentage of muted threads are failing.

I have manually correlated a few of the threads that should have been unsubscribed by visiting the Issue or PR on GitHub, and I haven't found any where I was still subscribed (suggesting that despite the failure message, I am being unsubscribed).

The other issue is that threads are no longer being 'unmuted' at all. A search for "is:mute" in gmail will show all of the threads I have muted. Some of them have new notifications because I was @mentioned back into the thread, but since they weren't unmuted I didn't see the notification come in.

**Comment by mastahyeti** [comment]
I've noticed this too, but haven't dug into it yet.

**Comment by mastahyeti** [comment]
Seems similar to [this issue](https://code.google.com/p/google-apps-script-issues/issues/detail?id=2758). They claim to have made some improvement a few weeks ago to address this.

**Comment by bwestover** [comment]
Let me know if there is any way I can help. I really like this feature 😸 

> They claim to have made some improvement a few weeks ago to address this.

Interesting, the timeline given there:
>As of May 25th, we've made several changes that should significantly reduce the number of timeout errors and latency in calling UrlFetch from a trigger.

Thats pretty close to when I STARTED having problems 😖

**Comment by mastahyeti** [comment]
I don't have much time to look into this. If you want to comment on that thread, I'd appreciate it.

**Comment by bwestover** [comment]
Sure thing. That is done. 👍

**Comment by bwestover** [comment]
Just wanted to give an update here. I had eventually turned off Muter due to the daily error emails.

9 days ago, I decided to try turning it back on. I've been using it every day since and haven't gotten a single error email :tada: 

I did incorporate @jasonrudolph's patch (https://github.com/jasonrudolph/OctoGAS/commit/75990fbc4f1b23d9fcd556f6112a8a4f6642e828) into my copy of the Muter script. I think this is a good idea, but I don't think its responsible for the lack of errors.
